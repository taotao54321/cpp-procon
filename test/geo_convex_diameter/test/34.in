23
-99.989887 -96.536848
-99.785114 -99.994406
98.992692 -99.953115
99.757393 -98.808188
99.914563 -94.709536
99.995842 -91.520692
99.998660 31.201036
99.935165 83.899473
99.851940 92.540166
99.687867 98.107691
97.545501 99.689055
94.436860 99.782267
89.111914 99.903393
71.478880 99.958013
42.810076 99.967412
-44.133729 99.989325
-69.110357 99.949735
-92.701661 99.769034
-99.337052 99.259933
-99.699190 99.201234
-99.781539 97.827025
-99.960717 64.374878
-99.987528 -70.020426
