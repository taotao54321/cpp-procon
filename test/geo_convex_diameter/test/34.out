281.319957874154
